upgrade/upgrade-scripts/verify-jar

```diff
@@ -26,12 +26,22 @@ function usage() {
 	exit 2
 }
 
-function run_support_info() {
+function verify_jar_verify_cleanup() {
 	local rc="$?"
 
-	/opt/delphix/server/bin/support_info \
-		-o "${LOG_DIRECTORY}/${CONTAINER}" \
-		-t "log dropbox" || die "Failed to collect verify logs."
+	#
+	# This name is used by the "upgrade-verify.jar" just executed, so we
+	# cannot change this value without also modifying that JAR.
+	#
+	MDS_SNAPNAME="MDS-CLONE-upgradeverify"
+
+	[[ "$(/opt/delphix/server/bin/dx_manage_pg isrunning -s ${MDS_SNAPNAME})" -eq 0 ]] &&
```

Comment:
This is just a style comment, but the `[[` isn't necessary.. e.g.
```
/opt/delphix/server/bin/dx_manage_pg isrunning -s "$MDS_SNAPNAME" &&
```
should work the same.

For all of the `if` conditionals, I don't think you need to use `[[ ... -eq 0 ]]`, you can just use the command with `if` and/or `&&` directly.

I'm fine either way, I think I'm just accustomed to not seeing `[[` in lots of our other scripts (e.g. in the app-gate).